src/integration-test/tests/citizen/defence/steps/defence.ts

```diff
@@ -260,6 +261,10 @@ export class DefenceSteps {
     defendantSteps.selectTaskFreeMediation(defendantType)
   }
 
+  askForHearingRequirements (defendantType: PartyType = PartyType.INDIVIDUAL): void {
```

Comment:
can we remove this please and just put it inline as it's only one line of code and doesn't really need to be in a separate function.

Reply:
I think it's alright to be like this because its used in 4 places.